cmd/abci-cli/abci-cli.go

```diff
@@ -295,6 +310,46 @@ func cmdCommit(c *cli.Context) error {
 	return nil
 }
 
+// Delineate beginning of block with header and hash
+func cmdBeginBlock(c *cli.Context) error {
+	args := c.Args()
+	if len(args) > 2 {
+		return errors.New("Command begin block takes max 2 arguments")
+	}
+	headerBytes, err := stringOrHexToBytes(c.Args()[0])
```

Comment:
You should check that you have at least one arg, else this could panic (index out of bounds)